Examples/Io/Performance/ActsExamples/Io/Performance/CKFPerformanceWriter.hpp

```diff
@@ -57,6 +57,10 @@ class CKFPerformanceWriter final : public WriterT<TrajectoriesContainer> {
     size_t nMeasurementsMin = 9;
     /// Min transverse momentum
     double ptMin = 1_GeV;
+    /// flag to use neural network for track classification
+    bool useMLTrackClassifier = false;
```

Comment:
I do not think you need the `useMLTrackClassifier` flag here. The function reference below can be used for that. By default, the `std::function` is initialized w/ `nullptr` and does not reference a valid function. You can check in the algorithm implementation whether a predictor is set via
```cpp
if (m_cfg.duplicatedPredictor) {
  ...
}
```
directly. This also avoids the corner case, where the `useMLTrackClassifier` flag is set, but the function is not actually set.